scaladoc/src/dotty/tools/scaladoc/tasty/SyntheticSupport.scala

```diff
@@ -75,11 +75,7 @@ object SyntheticsSupport:
     given ctx: dotc.core.Contexts.Context = quotes.asInstanceOf[scala.quoted.runtime.impl.QuotesImpl].ctx
     val sym = rsym.asInstanceOf[dotc.core.Symbols.Symbol]
     // `lookupPrefix` is private in `QuotesImpl#SymbolMethods`
-    val lookupPrefix =
-      if sym.isClass then
-        sym.thisType
-      else
-        sym.namedType
+    val lookupPrefix = sym.namedType
```

Comment:
This method is copied from `QuotesImpl` and I'm not sure if we should change it. Do you have any explanation there? Also, if we change the implementation there, we should remove the comment above

Reply:
The explanation is in `QuotesImpl`:
```
private def lookupPrefix: TypeRepr =
  if self.isClass then
    self.thisType // Needed to handle self-types (as in tests/run-macros/self)
  else
    self.namedType
```
Ok, will change the comment.